react_juce/core/TextInputView.h

```diff
@@ -0,0 +1,118 @@
+/*
+==============================================================================
+
+  TextInputView.h
+  Created: 20 Jan 2021 10:30pm
+
+==============================================================================
+*/
+
+#pragma once
+
+#include "View.h"
+
+namespace blueprint
+{
+    //==============================================================================
+    /** The TextInput class is an extended juce::TextEditor class for TextInputView
+     * class. There are some features added.
+     *
+     * - Invoke JavaScript's 'input' and 'change' event.
+     *
+     *   Note: React.js has only 'onChange' callback which has same behavior as
+     *   JavaScript's 'onInput' and does not have any callback corresponding to
+     *   original JavaScript's 'onChange'.
+     *   https://stackoverflow.com/questions/38256332/in-react-whats-the-difference-between-onchange-and-oninput
+     *
+     * - Support React's controlled component model.
+     *   https://reactjs.org/docs/uncontrolled-components.html
+     *   If field `controlled` is true,
+     *   only the string value given by `setControlledValue()`
+     *   is shown in the text editor.
+     *
+     * - Allow to set 'placeholder' text and 'placeholder-color' separately.
+     */
+    class TextInput : public juce::TextEditor, public juce::TextEditor::Listener
+    {
+    public:
+        //==============================================================================
+        explicit TextInput(const juce::NamedValueSet *_props)
```

Comment:
Is it possible for this to be made a reference? I might be missing something obvious!

Reply:
Thanks @JoshMarler for reviewing! I think it's possible😁
Do you want me to change the member `const juce::NamedValueSet *props` to `const juce::NamedValueSet &props` too?

Reply:
Yep perfect

Reply:
I fixed it in c724958.